assets/js/translator.js

```diff
@@ -52,6 +52,18 @@ if(modeEnabled('translation')) {
                 synchronizeTextareaHeights();
             });
 
+            $('#clearText').click(function () {
+                $('#translatedText').val('');
+                $('#originalText').val('');
+                synchronizeTextareaHeights();
+            });
+
+            $('#copyText').click(function () {
+                $('#translatedText').select();
+                document.execCommand('copy');
+                synchronizeTextareaHeights();
```

Comment:
Why does copying text require this?

Reply:
This is the functionality to copy the text from translatedText field.

Reply:
Select + `execCommand('copy')` is the normal way to copy text. There is a HTML5 API, but browser support is basically non-existent so far. Though `synchronizeTextareaHeights()` probably isn't needed here - can't see why copying text would cause a height change.

Reply:
Yep, I was referring to the extraneous `synchronizeTextareaHeights()`.